custom/icds_reports/utils/aggregation_helpers/distributed/growth_monitoring_forms.py

```diff
@@ -183,10 +190,7 @@ def aggregation_query(self):
           FROM ({ucr_table_query}) ucr
           FULL OUTER JOIN "{tablename}" prev_month
           ON ucr.case_id = prev_month.case_id AND ucr.supervisor_id = prev_month.supervisor_id
-            AND ucr.month::DATE=prev_month.month + INTERVAL '1 month'
-          WHERE coalesce(ucr.month, %(month)s) = %(month)s
-            AND coalesce(prev_month.month, %(previous_month)s) = %(previous_month)s
-            AND coalesce(prev_month.state_id, %(state_id)s) = %(state_id)s
```

Comment:
@snopoke FYI that I see these coalesce statements were added in https://github.com/dimagi/commcare-hq/commit/580e6930634c#diff-c978e92b666a2ca78f81cad75ea130e4R183 At that point in the commit history they were also visible in the monolith queries. They are no longer in they monolith queries https://github.com/dimagi/commcare-hq/blob/d4340b2496f9858da9983a64773656c12418e2bf/custom/icds_reports/utils/aggregation_helpers/monolith/growth_monitoring_forms.py#L177 and I cannot find where they were added or removed between several different directory moves.

I'm pretty sure this query is safe, but wanted to flag in case you have any concerns or memories of why they were added

Reply:
I see now that this is for the case when the previous month's records do not exist yet